index.html

```diff
@@ -189,16 +199,37 @@ <h2>Languages</h2>
                     </div>
                   </div>
                   <div class="Frame flex-row">
-                    <p>Frameworks</p>
+                    <h1 class="framework">Frameworks</h1>
                     <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
                   </div>
                   <div class="skill flex-row">
-                    <p>Skills</p>
+                    <h4>Skills</h4>
                     <i class="fa fa-angle-right fa-2x" aria-hidden="true"></i>
                   </div>
                 </div>
               </section>
         </section>
+
+        <section>
+          <div class="contact-form">
+            <div class="contact-writeup">
+              <h1>Contact me</h1>
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding.
+                 I'd love to help with it</p>
+            </div>
+
+            <form>
+              <div class="contact-form-div">
+                <input type="text" placeholder="enter name" class="contact-input" maxlength="30" required>
+                <input type="text" placeholder="enter email" class="contact-input" required>
+                <textarea name="" id="" cols="45" rows="10" class="contact-text" placeholder="write your message here"
```

Comment:
- [ ] your design for the textarea which holds the message is currently overflowing the viewport and as such it is affecting the overall design of your web page. 
- [ ] This is happening because you are making use of html `cols & row` to style the textarea however i would suggest you make use of proper css styling to correctly design your web page. :+1:  :sparkles: 
- your design
![te-a](https://github.com/Developer-Kingz/my-portforlio-project/assets/89055502/dd6cb65d-875b-4520-8e6f-721719d5da35)